parts/kubernetesmaster-kube-apiserver.yaml

```diff
@@ -29,6 +29,9 @@ spec:
         - "--tls-private-key-file=/etc/kubernetes/certs/apiserver.key"
         - "--client-ca-file=/etc/kubernetes/certs/ca.crt"
         - "--service-account-key-file=/etc/kubernetes/certs/apiserver.key"
+        - "--oidc-client-id=<oidcClientId>"
+        - "--oidc-issuer-url=<oidcIssuerUrl>"
+        - "--oidc-username-claim=oid"
```

Comment:
oidc Auth is enabled when both 'oidc-client-id' and 'oidc-issuer-url' are not null:
https://github.com/kubernetes/kubernetes/blob/b78fc209a49049312ae038ef0235141f6c6eb929/pkg/kubeapiserver/authenticator/config.go#L152

For api model without AAD profile, those value would be empty, so that oidc auth is disabled:
https://github.com/Azure/acs-engine/pull/1049/files#diff-b7e839779245b296d836bd5663a27941R272